### PR TITLE
User should be able to login

### PR DESCRIPTION
**Pull request by perhegg**
#### Description
Changes proposed in this pull request:

* Add login form 
* Add login functionality
* Use local storage
#### What I have learned working on this feature: 
How to mock api responses

#### Screenshots:
image

image

### CODE REVIEW

File: src/kimaiService.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      .then(response => {
+				console.log(response.data)
+				resolve({message: "Successfull"});
+      })
+      .catch(error => {
+				debugger
+        resolve(error.response.data)
+      })
+  });
+};
```
Comment (vick3d) [inline src/kimaiService.js]: should this file maybe be in the src/modules folder? (as the previous merge of timeTrackingTable)

File: src/components/loginForm.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+					<Form size='large' >
+						<Segment stacked>
+							<Form.Input value={userName} onInput={e => setUserName(e.target.value)} fluid icon='user' iconPosition='left' placeholder='Username' />
+							<Form.Input
+								value={password} onInput={e => setPassword(e.target.value)}
+								fluid icon='lock'
+								iconPosition='left'
+								placeholder='Password'
+								type='password'
+							/>
```
Comment (amandagibson) [inline src/components/loginForm.js]: move this up?

File: src/kimaiService.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    axios
+      .get(apiUrl, {
+        headers: headers
+      })
+      .then(response => {
+				console.log(response.data)
+				resolve({message: "Successfull"});
+      })
+      .catch(error => {
+				debugger
```
Comment (amandagibson) [inline src/kimaiService.js]: remove this

File: src/kimaiService.js (hunk 1 above)
Comment (perhegg) [reply src/kimaiService.js]: done

File: src/components/loginForm.js (hunk 2 above)
Comment (perhegg) [reply src/components/loginForm.js]: done

File: src/kimaiService.js (hunk 3 above)
Comment (perhegg) [reply src/kimaiService.js]: done

**Pull request merged**